_rules/SC2-4-4-link-has-name.md

```diff
@@ -0,0 +1,167 @@
+---
+name: Links have an accessible name
+description: |
+  Each link has an accessible name
+
+success_criterion:
+- 4.1.2 # Name, Role, Value
+
+test_aspects: # Remove what is not applicable
+- DOM Tree
+- CSS Styling
+
+authors:
+- Wilco Fiers
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML or SVG element with the [semantic role](#semantic-role) of `link` that is [visible on the page](#visible-on-the-page) or [exposed to assistive technologies](#exposed-to-assistive-technologies).
```

Comment:
There are no SVG tests. I suggest we remove this part and tackle SVG at a later stage.